vocode/streaming/models/synthesizer.py

```diff
@@ -160,13 +160,20 @@ def override_voice_id_with_prompt(cls, voice_id, values):
 
 
 PLAYHT_DEFAULT_VOICE_ID = "larry"
+PLAYHT_DEFAULT_QUALITY = "medium"
+PLAYHT_DEFAULT_OUTPUT_FORMAT = "mp3"
+PLAYHT_DEFAULT_SPEED = 1
+PLAYHT_DEFAULT_RATE = 24000
 
 
 class PlayHtSynthesizerConfig(SynthesizerConfig, type=SynthesizerType.PLAY_HT.value):
     voice_id: str = PLAYHT_DEFAULT_VOICE_ID
-    speed: Optional[str] = None
-    preset: Optional[str] = None
-    quality: Optional[str] = None
+    quality: Optional[str] = PLAYHT_DEFAULT_QUALITY
+    output_format: Optional[str]  = PLAYHT_DEFAULT_OUTPUT_FORMAT
```

Comment:
do we have support for other output formats in the synthesizer? it always needs to output wav in the end (that's why we have that `AudioSegment.from_mp3` call